packages/web/src/scss/components/Accordion/_Accordion.scss

```diff
@@ -0,0 +1,90 @@
+@use 'sass:map';
+@use 'theme';
+@use '../../tools/typography';
+
+.Accordion__itemHeader {
+    position: relative;
+    display: flex;
+    gap: theme.$accordion-header-gap;
+    align-items: flex-start;
+    justify-content: space-between;
+    width: 100%;
+    height: auto;
+    padding: theme.$accordion-header-padding-y theme.$accordion-header-padding-x;
+    margin-bottom: 0;
+    border-radius: theme.$accordion-border-radius;
```

Comment:
you are missing the background color here. It is set in figma.

Reply:
set